app/views/steps/_hidden_prisoner_step.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= hidden_field_tag('prisoner_step[first_name]',
+                     @prisoner_step.first_name) %>
+<%= hidden_field_tag('prisoner_step[last_name]',
+                     @prisoner_step.last_name) %>
+<%= hidden_field_tag('prisoner_step[date_of_birth]',
+                     @prisoner_step.date_of_birth) %>
+<%= hidden_field_tag('prisoner_step[number]',
+                     @prisoner_step.number) %>
+<%= hidden_field_tag('prisoner_step[prison_id]',
+                     @prisoner_step.prison_id) %>
```

Comment:
I'm torn between a desire to DRY this up and a general distaste of gratuitous metaprogramming